docs/rules/max-lines-per-function.md

```diff
@@ -46,7 +46,7 @@ This rule has a number or object options:
 
 * `"IIFEs": true` include any code included in IIFEs.
 
-Optionally, you may specify an singular integer for the `max` option:
+Alternatively, you may specify an single integer for the `max` option:
```

Comment:
This is looking a lot better, but we should still replace "an" with "a" here. (I think maybe you had written "an integer" at one point, then added "singular"/"single"?)